scripts/userstyles.yml

```diff
@@ -187,27 +187,27 @@ userstyles:
       past-maintainers: [*rubyowo]
   chatreplay:
     name: ChatReplay
-    category: leisure
+    categories: [entertainment, social_networking]
     color: peach
     readme:
       app-link: "https://chatreplay.stream"
       current-maintainers: [*anubisnekhet]
   chess.com:
     name: Chess.com
-    category: game
+    categories: [game]
```

Comment:
Chess.com also offers education services, as a 2nd category we should probably put it down.